### DISCUSSION TITLE
What about adding "sharpen" to the bundle

### DISCUSSION THREAD

**Discussion by c33s** [liip/LiipImagineBundle#1419] [Ideas]
recently i needed a sharpen filter in my project and noticed that this bundle doesn't provide one.
as the code was quite short, i asked myself if this couldn't be bundled in this package to create a better DX. maybe even add things as `brightness`, `blur`, `colorize`, `convolve`, `gamma` and `negative` as this methids exists on the `EffectInterface`. 

i haven't done a lot of digging if the interface is implemented on all image processing providers but even if it is only supported partially it would be a win.

what do you think?

```
<?php

declare(strict_types=1);

namespace App\Imagine\Filter\Loader;

use Imagine\Image\ImageInterface;
use Liip\ImagineBundle\Imagine\Filter\Loader\LoaderInterface;

final class SharpenFilter implements LoaderInterface
{
    /**
     * @param array<mixed> $options
     */
    public function load(ImageInterface $image, array $options = []): ImageInterface // phpcs:ignore
    {
        $image->effects()->sharpen();

        return $image;
    }
}
```

**Comment by dbu** [comment]
hi, thanks for the input. as long as imagine offers a filter, i agree it would be expected that the bundle offers a way to add the filter to the stack. happy if you want to do pull requests. i suggest to do one request per new filter to keep the merge more reasonable. when you add a filter, please don't forget to also add documentation for it.
